bin/run.sh

```diff
@@ -34,8 +34,13 @@ elif [ ! -e Cargo.lock ]; then
     echo "WARNING: student did not upload Cargo.lock. This may cause build errors." | tee -a "$output_path/results.out"
 fi
 
+release=""
+if jq --exit-status '.custom?."test-in-release-mode"?' "$solution_path"/.meta/config.json; then
```

Comment:
Have you found documentation showing that this is where the config file is injected into the test runner? Looking up the actual injected location is probably the single hardest part of this PR.

Reply:
I think I don't understand how Exercism works to answer this question. I thought that this was checked in for each exercise and is "injected" with everything else except files provided by the student.

Reply:
Basically, what happens is that the test runner will get the entire contents of the exercise directory as listed in the rust repo, but without the `.docs` folder and with the student's submitted files replacing existing files. So `.meta/config.json` is indeed the right path.